schema.sql

```diff
@@ -6,5 +6,9 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(100)
 );
+
```

Comment:
- [ ] `README.md` file: I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

- [ ] Kindly make sure you update your `# Getting Started` section by adding there all information needed such as about to `clone`, `setup`, `use` your repo, `tools needed`,,,, This will make your documentation be descriptive and professional. I suggest you refer this **[example](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** to help you

Reply:
Thank you @tresorsawasawa for the review comment.This has been resolved 👏 using the new [template-databases](https://github.com/microverseinc/curriculum-template-databases/blob/main/README.md)